client/lib/preferences-helper/boolean-preference.tsx

```diff
@@ -45,18 +47,18 @@ const BooleanPreference: FunctionComponent< Props > = ( { name, value } ) => {
 			{ value !== localValue && (
 				<>
 					<button
-						className="preferences-helper__save-pref-button"
+						className="preferences-helper__preference-save"
```

Comment:
Does the `__preference-save` need to always contain the `preference` word? It's already in the prefix, so the class names seem too wordy to me.